README.md

```diff
@@ -62,6 +62,8 @@
 
         composer require dragonmantank/cron-expression
 
+5. Till `symfony/recipes-contrib` is updated for the v3, you must add `sylius_scheduler_command.yaml` from `install/Application/config/{packages,routes}` to your project by respecting the same folder architecture.
```

Comment:
IMHO, you should provide bash commands to copy them. It would be easier for users. And some project don't use `symfony/flex`